common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -108,8 +108,12 @@ public int numConnectionsPerPeer() {
     return conf.getInt(SPARK_NETWORK_IO_NUMCONNECTIONSPERPEER_KEY, 1);
   }
 
-  /** Requested maximum length of the queue of incoming connections. Default is 64. */
-  public int backLog() { return conf.getInt(SPARK_NETWORK_IO_BACKLOG_KEY, 64); }
+  /**
+   * Requested maximum length of the queue of incoming connections. If  &lt; 1,
```

Comment:
@xCASx . This PR has `&lt; 1` here (line 112) and `-1` at line 114. Shall we make it consistent?

Reply:
[Here is how it implemented](https://github.com/apache/spark/blob/09ed64d795d3199a94e175273fff6fcea6b52131/common/network-common/src/main/java/org/apache/spark/network/server/TransportServer.java#L117):
```java
if (conf.backLog() > 0) {
  bootstrap.option(ChannelOption.SO_BACKLOG, conf.backLog());
}
```
I've been thinking about wording. If use `0 or negative` instead of `< 1` it may seem that separately mentioned `0` is some special case different to `negative` values.

For me it's not a big deal, if you'd like, I can change it to `0 or negative`.

Reply:
What I meant was `If < 1` looks strange grammatically to me.